contrib/prometheus/Dockerfile

```diff
@@ -1,28 +1,31 @@
-FROM arm64v8/alpine:3.5
+FROM alpine:3.6
```

Comment:
Do we need to build prom for non-armhf platforms still?

Reply:
Yes - there is no official image.